code/game/objects/items/stacks/medical.dm

```diff
@@ -12,6 +12,8 @@
 	max_integrity = 40
 	novariants = FALSE
 	item_flags = NOBLUDGEON
+	cost = 250
```

Comment:
So you're moving the (borg) fab cost to the item itself, rather then a subtype, is that right?

Reply:
Yes.